contrib/buildrpm/README

```diff
@@ -0,0 +1,59 @@
+
+# Copyright (c) 2016      Cisco Systems, Inc.  All rights reserved.
+#
+# Written by: Landon Curt Noll, Martin Kontsek
+#
+# Permission to use, copy, modify, and distribute this software and
+# its documentation for any purpose and without fee is hereby granted,
+# provided that the above copyright, this permission notice and text
+# this comment, and the disclaimer below appear in all of the following:
+#
+#       supporting documentation
+#       source copies
+#       source works derived from this source
+#       binaries derived from this source or from derived source
+#
+# Landon Curt Noll, Martin Kontsek DISCLAIMS ALL WARRANTIES WITH REGARD
+# TO THIS SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+# AND FITNESS. IN NO EVENT SHALL Landon Curt Noll AND Martin Kontsek BE
+# LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY
+# DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+
+=================================================================
+
+Script "buildrpmLibfabric.sh" can be used to build RPM from the
+stable Libfabric source tarball. Script was tested on RedHat 7.
+
+You need to supply one mandatory parameter to the script:
+filename of the Libfabric tarball.
+There are several voluntary parameters available:
+
+-n
+    Do nothing, useful with -v option. If used with -v option,
+    it will just print what would have been done.
+-o
+    Install path of the RPM would be changed to
+    /opt/libfabric/_VERSION_. Default install
+    path of the RPM is /usr
+-m
+    Modulefile with .tcl extension will be created in rpmbuild
+    directory. By default, modulefile is not created.
+-u
+    Build with usNIC support only. By default, support for
+    usNIC, Infiniband, Sockets, etc. will be built.
```

Comment:
Can we get something more generic, like adding a parameter as to which provider to build? This options seems goofy.